omni-slider.js

```diff
@@ -200,6 +202,17 @@ Slider.prototype.init = function (options) {
   }
 };
 
+Slider.prototype._applyCallback_ = function (data, callback) {
+
```

Comment:
:) I use whitespace to separate my block statement outer line, and it's inner line(s). I guess that style is not compiled with this repo's style. I guess I will shrink all the white spaces to make this Pull Request complied.